pkg/apis/resource/types.go

```diff
@@ -652,19 +643,10 @@ type ResourceClaimStatus struct {
 	// +patchMergeKey=uid
 	ReservedFor []ResourceClaimConsumerReference
 
-	// Indicates that a claim is to be deallocated. While this is set,
-	// no new consumers may be added to ReservedFor.
-	//
-	// This is only used if the claim needs to be deallocated by a DRA driver.
-	// That driver then must deallocate this claim and reset the field
-	// together with clearing the Allocation field.
-	//
-	// This is an alpha field and requires enabling the DRAControlPlaneController
-	// feature gate.
-	//
-	// +optional
-	// +featureGate=DRAControlPlaneController
-	DeallocationRequested bool
+	// DeallocationRequested is tombstoned since 1.32 since Kubernetes 1.32 where
```

Comment:
Two "since" clauses

Reply:
What can I say? At least I copied it consistently everywhere! :sweat_smile: 

[Fixed](https://github.com/kubernetes/kubernetes/compare/c7fac99c4322c82e2f0d19a379fc6c256f67d6cf..c6b172d1a6f0620feb1c2368ad82710ce328350a).

Is this ready for approval?

We had thumbs up from:
- [John](https://github.com/kubernetes/kubernetes/pull/128003#pullrequestreview-2369796234)
- [Kevin](https://github.com/kubernetes/kubernetes/pull/128003#pullrequestreview-2366558468)
- [Anish](https://github.com/kubernetes/kubernetes/pull/128003#issuecomment-2411698045) for SIG Auth

/assign @alculquicondor 

For SIG Scheduling.

Reply:
Also from [Tim](https://github.com/kubernetes/kubernetes/pull/128003#pullrequestreview-2370376213).

Reply:
And [Aldo](https://github.com/kubernetes/kubernetes/pull/128003#issuecomment-2417422668).

@thockin: I think it's okay for you to LGTM again (just small edits) and do the (probably) final approval.